azext_iot/tests/test_iot_ext_int.py

```diff
@@ -1437,9 +1437,51 @@ def test_storage(self):
         )
 
     @pytest.mark.skipif(
-        not LIVE_IDENTITY, reason="azext_iot_testidentity env var not set"
+        not LIVE_STORAGE_ID or not LIVE_STORAGE,
```

Comment:
what about `not all([LIVE_STORAGE_ID, LIVE_STORAGE])` for the skip condition -- when you need both `LIVE_STORAGE_ID` and `LIVE_STORAGE` to have values.